en/components/for_of.md

```diff
@@ -212,6 +212,7 @@ In the example above, an `even` class is assigned to every even div element.
 |Limitation|Description|
 |--- |--- |
 | Scroll position of components that use `igxForOf` is not preserved in projected container | When a component that uses `igxForOf` directive is in a `<ng-content>`, or other projected container, its scrollbar position won't be preserved, even though the data position would be. That's because the DOM elements are detached and later re-attached to the DOM tree and as a result, losing the scrollbar scroll position. The affected controls that use `igxForOf` internally are: `igxGrid`, `igxHierarchicalGrid`, `igxTreeGrid`, `igxCombo`. <br/>Some possible workarounds are: <br/> <ul><li>Resetting the DOM node's state, for example by wrapping it in an `ngIf`. The component will be destroyed and then re-created, losing all internal states;</li><li>Persisting the state, i.e. determining whether the element has been re-added to the DOM tree and resetting its scroll position manually, for example using `MutationObserver`</li></ul>
+| Content size changes at runtime are not anticipated by the `igxForOf` | When an element inside the `igxForOf` resizes at runtime, the recalculated width/height positions of the subsequent elements would not be in compliance with their actual ones. That behavior occurs because the directive renders and adjusts the scroll positions at once. In case any content size changes occur after that, they will not be taken into account.<br>A possible workaround of that problem is to set the dimensions of the projected container explicitly, so that any further resizes won't affect neither the display of the content, nor the calculations, performed by the `igxForOf`.
```

Comment:
Issue should be specific about "scrollTo" method as everything else works as expected:

`scrollTo` method does not work correctly when the content size of the rendered templates changes post initialization . |  When the elements inside the template have size that changes runtime after initialization (for example as a result of content projection, remote request resolution etc.) then the `scrollTo` method will not be able to scroll to the correct index. The method will scroll to the position of the index before the runtime size change occurs, hence the location will not be correct after the size is changed later. A possible workaround is to use templates that do not change their size based on their content if the content is loaded later.